### PR TITLE
Msw update from 1.2.1 to 2.0.1

### PR DESCRIPTION
**Pull request by seenanair**
msw package updated from 1.2.1 to 2.0.1

The main changes are

Call signature changes for graphql mutationn and query ( breaking change from 2.0 - Response resolver call signature is no longer (req, res, ctx) => res() but instead ({ request }) => new Response().)
setupWorker call chnaged to msw/browser
https://github.com/mswjs/msw/releases?page=2
https://mswjs.io/docs/api/graphql/

### CODE REVIEW

**Review by khelwood — APPROVED** [review]
For this kind of thing in the future, it would be good to write stuff in such a way that the library-specific code is inside a function so that the change can just be made in one place instead of having to change the code everywhere it is used.

File: cypress/e2e/pages/visiumTransfer.cy.ts [hunk 1]
```diff
@@ -275,22 +279,6 @@ describe('Transfer Page', () => {
         });
       });
 
-      context('when user selects a mapped slot', () => {
```
[the remark this answers is not in the document]
Comment (seenanair) [reply cypress/e2e/pages/visiumTransfer.cy.ts:278]: This test is to check whether a  selection of a slot displays the mappings for that selected slot in a table. But, this is already covered in  above section where context('when user maps slots in many to one mode') -> it('displays the table with A2, B2 slots mapped to D1')

File: cypress/e2e/pages/labwareDetails.cy.ts [hunk 2]
```diff
@@ -46,14 +37,13 @@ describe('Labware Info Page', () => {
     });
   });
 
-  context('when I check a flagged labware details', () => {
```
[the remark this answers is not in the document]
Comment (seenanair) [reply cypress/e2e/pages/labwareDetails.cy.ts:49]: Well spotted. This is a mistake. I have added it back in.

**Comment by seenanair** [comment]
> For this kind of thing in the future, it would be good to write stuff in such a way that the library-specific code is inside a function so that the change can just be made in one place instead of having to change the code everywhere it is used.

Yes, I totally agree and I thought about while doing this. I think probably the way to go is  a generic function where the library specifics like request and response object can be defined within , which can wrap the required context specific response.

**Review by sabrine33 — APPROVED** [review]

**Pull request merged**